packages/components/src/heading/stories/index.story.tsx

```diff
@@ -10,7 +10,8 @@ import { Heading } from '..';
 
 const meta: Meta< typeof Heading > = {
 	component: Heading,
-	title: 'Components (Experimental)/Heading',
+	title: 'Components (Experimental)/Typography/Heading',
+	id: 'components-heading',
```

Comment:
As this component is still formally experimental, if we change its ID, it will cause the previous URL not to work:

http://localhost:50240/?path=/docs/components-experimental-heading--docs

To fix that, we have 2 options:
- Use `id: 'components-experimental-heading'`
- Add the component to `PREVIOUSLY_EXPERIMENTAL_COMPONENTS`. 

Since the component is still experimental, I'd prefer the former option. Once we stabilize the component, we can always add it to `PREVIOUSLY_EXPERIMENTAL_COMPONENTS`